packages/mobile/src/components/lineup/types.ts

```diff
@@ -41,6 +43,17 @@ export type LineupProps = {
    */
   disableTopTabScroll?: boolean
 
+  /**
+   * Optional payload to pass to the fetch action
+   */
+  fetchPayload?: any
```

Comment:
no worries but could be good to have some typing, though now that i see it's use, not sure how you could